docs/openapi.md

```
@@ -5187,6 +5344,62 @@ SQLite database configuration.
+
+Options for ranking and filtering search results.
+
+This class configures how search results are ranked and filtered. You can use algorithm-based
+rerankers (weighted, RRF) or neural rerankers. Defaults from VectorStoresConfig are
+used when parameters are not provided.
+
+Examples:
+    # Weighted ranker with custom alpha
+    SearchRankingOptions(ranker="weighted", alpha=0.7)
+
+    # RRF ranker with custom impact factor
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)